torch/_torch_docs.py

```diff
@@ -5897,6 +5897,55 @@ def merge_dicts(*dicts):
     tensor([  435.,  1335.,  2235.,  3135.])
 """.format(**multi_dim_common))
 
+add_docstr(torch.nansum,
+           r"""
+nansum(input, dtype=None) -> Tensor
+
+Return the sum of array elements over a given axis treating Not a Numbers (NaNs) as zero.
```

Comment:
It's not "over a given axis" for this variant, right? This just takes the sum of the entire tensor?

Reply:
FIxed. Thanks!